src/Symfony/Bridge/Monolog/Formatter/ConsoleFormatter.php

```diff
@@ -101,8 +102,11 @@ public function formatBatch(array $records): mixed
     /**
      * {@inheritdoc}
      */
-    public function format(array $record): mixed
+    public function format(array|LogRecord $record): mixed
```

Comment:
This change is technically a BC break because the method is not final and thus could be overridden downstream.